third_party/xla/xla/tools/BUILD

```diff
@@ -7,6 +7,15 @@ load(
     "if_cuda_or_rocm",
 )
 load("@local_config_cuda//cuda:build_defs.bzl", "if_cuda")
+load(
+    "@local_tsl//tsl/platform/default:cuda_build_defs_bzl",
+    "if_cuda_is_configured",
+)
+load(
+    "@local_config_rocm//rocm:build_defs.bzl",
+    "if_rocm_is_configured",
+)
```

Comment:
this `if_rocm_is_configured` might not need anymore as `if_cuda_or_rocm` is used in this BUILD.